drivers/wifi/eswifi/eswifi_core.c

```diff
@@ -646,7 +646,7 @@ static int eswifi_init(struct device *dev)
 	}
 	eswifi->resetn.pin = DT_INVENTEK_ESWIFI_ESWIFI0_RESETN_GPIOS_PIN;
 	gpio_pin_configure(eswifi->resetn.dev, eswifi->resetn.pin,
-			   GPIO_DIR_OUT);
+			   GPIO_OUTPUT);
```

Comment:
We should also pull in DTS configuration from `DT_INVENTEK_ESWIFI_ESWIFI0_RESETN_GPIOS_FLAGS` as in
```
			   GPIO_OUTPUT_INACTIVE | DT_INVENTEK_ESWIFI_ESWIFI0_RESETN_GPIOS_FLAGS);
```